### PR TITLE
Register my.mr-julus.is-a.dev

### PR DESCRIPTION
**Pull request by BougSoon-Studio**
Register my.mr-julus.is-a.dev with CNAME record pointing to my-mr-julus.web.app

### CODE REVIEW

File: domains/my.mr-julus.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Personal WebSite",
+  "owner": {
+    "username": "BougSoon-Studio",
+    "email": "bougsoon.studio@gmail.com"
+  },
+  "record": {
+    "CNAME": "my-mr-julus.web.app"
```
Comment (CuteDog5695) [inline domains/my.mr-julus.json:8]: Site not found, please give a preview of your website.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/my.mr-julus.json (hunk 1 above)
Comment (BougSoon-Studio) [reply domains/my.mr-julus.json:8]: The site is now avaible ;)

Comment (BougSoon-Studio) [reply domains/my.mr-julus.json:8]: This is a preview of the website :
Capture d’écran 2024-08-31 à 14 47 01

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**